src/coreclr/jit/emitriscv64.cpp

```diff
@@ -3659,6 +3684,50 @@ void emitter::emitDisInsName(code_t code, const BYTE* addr, instrDesc* id)
             }
             return;
         }
+        case 0b0101111: // AMO - atomic memory operation
```

Comment:
Could you update to hex?

Reply:
Sure, I'll leave the funct5 binary though because it doesn't fall on the nibble boundary and would be difficult to eye-match with hex codes in instrsriscv64.h.